bin/improver-nbhood

```diff
@@ -35,15 +35,18 @@ import argparse
 
 import iris
 
-from improver.nbhood import BasicNeighbourhoodProcessing
+from improver.nbhood import NeighbourhoodProcessing
 
 
 def main():
     """Load in arguments and get going."""
     parser = argparse.ArgumentParser(
-    description='Apply basic weighted circle smoothing via ' +
-                'the BasicNeighbourhoodProcessing plugin ' +
+    description='Apply the requested kernel via ' +
+                'the NeighbourhoodProcessing plugin ' +
                 'to a file with one cube.')
+    parser.add_argument('kernel_method', metavar='KERNEL_METHOD',
+                        help='The kernel method to apply in neighbourhood'
+                             ' processing. Options: "circular".')
     group = parser.add_mutually_exclusive_group()
```

Comment:
I've left the `Options: "circular"` description in the help message as it is, as I was actually intending for this to be updated every time a new kernel was added. I don't think that we'll be adding lots of kernels, but I think that it's useful to have the options documented at the top level, rather than relying upon the error message, which is generated if someone runs neighbourhood processing by passing an unsupported kernel.